docs/excel/custom-functions-overview.md

```diff
@@ -149,6 +149,25 @@ You can also use the following syntax:
 Office.Preview.StartCustomFunctions();
 ```
 
+## Handling errors
+Error handling for custom functions is the same as [error handling for the Excel JavaScript API at large](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-error-handling). Generally, you will use `.catch` to handle errors. The code below gives an example of `.catch()`. 
```

Comment:
Change this link to be relative. Also, avoid including en-us in links, as it breaks localization.